explore/src/main/webapp/main.jsx

```diff
@@ -15,10 +15,7 @@ import { registerSW } from "virtual:pwa-register";
 
 // Setup the Service Worker
 if ("serviceWorker" in navigator
-  // && !/localhost/.test(window.location) && !/local.lucuma.xyz/.test(window.location)) {
-    // Let's not do this in production yet
-  && !/local.lucumay.xyz/.test(window.location)
-  && !/explore.gemini.edu/.test(window.location) && !/gemini.edu/.test(window.location)) {
+  && !/local.lucumay.xyz/.test(window.location)) {
```

Comment:
Is this to avoid PWA in development? There is (and was) an extra `y` there, which leads me to believe this is not doing what we wanted it to do.

Reply:
ah interesting I'll fix it, to use pwa in dev you need to also setup the pwa plugin in dev mode so this maybe redundant